### PR TITLE
Blog about Data Science ready to be reviewed. 

### PR DESCRIPTION
**Pull request by claudiaflautero**
Hi, the blog is complete and ready for review.

### CODE REVIEW

File: _posts/2020-11-09-introduction_to_data_science.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Data is collected **everywhere**, and can be used with different purposes such as:
+
+- *Describe* the current state of things, organizations or a determined process.
+- *Diagnose* the cause of determined behaviors.
+- *Detect* events that can be described as anomalous or different from what the normal process is, like irregular credit car purchases.
+- *Predict* future events.
+
+## Why is Data Science so popular today?
+
+Everyday, around the world, data is being collected more than has ever been. Every transaction that a single person does is connected trough their email address, to their social media, what they like and dislike, where they have visited or are looking to visit, and what they can afford or not. All this information is valuable for companies and governments.
```
Comment (Alicebada) [inline _posts/2020-11-09-introduction_to_data_science.markdown]: Probably change this "Everyday, around the world, data is being collected more than has ever been" to "Every day, around the world, data is being collected more than ever"

Comment (Alicebada) [inline _posts/2020-11-09-introduction_to_data_science.markdown]: "Every transaction that a single person does is connected trough their email address, to their social media, what they like and dislike, where they have visited or are looking to visit, and what they can afford or not. All this information is valuable for companies and governments." can be rewritten as below;

"Every transaction a person makes is connected through their email address to their social media. Also, their preferences, past or upcoming vacation destinations, and what they can afford or not are equally recorded. All these pieces of information are available and valuable to companies and governments."

Please feel free to make changes as well.

File: _posts/2020-11-09-introduction_to_data_science.markdown [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+#### What happens after all this data is collected?
+
+After the raw data is collected and put together, is important to do an analysis to extract valuable information from it. The raw data is important, yes, but the multiple analysis and conclusions that can be drawn from there are far more interesting than just having all the data without processing.
```
Comment (Alicebada) [inline _posts/2020-11-09-introduction_to_data_science.markdown]: There is a typographical error; "After the raw data is collected and put together, is important" to
"After the raw data is collected and put together, it is important"

File: _posts/2020-11-09-introduction_to_data_science.markdown [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Introduction to Data Science!"
+date:   2020-11-09 13:30:00 -0700
+categories: Data Science
```
Comment (Alicebada) [inline _posts/2020-11-09-introduction_to_data_science.markdown]: You can add "raw data" and "data" to the categories also

**Review by Alicebada — CHANGES_REQUESTED** [review]
Hello Claudia,

Great job with the blog post. 

You just have some corrections to make which I have highlighted.

**Review by naupaka — CHANGES_REQUESTED** [review]
Looks good! Please address @Alicebada's suggestions and then I think this is ready to merge.

**Comment by claudiaflautero** [comment]
The changes were done to the blog. It should be ready to merge.

**Comment by naupaka** [comment]
👍

**Pull request merged**